sympy/ntheory/tests/test_elliptic_curve.py

```diff
@@ -18,3 +18,6 @@ def test_elliptic_curve():
     assert EllipticCurve(-2731, -55146, 1, 0, 1).discriminant == 25088
     # Torsion points
     assert len(EllipticCurve(0, 1).torsion_points()) == 6
+
+    e1 = EllipticCurve(-17, 16, modulus = 307)
+    assert e1.__contains__([7, 213, 1]) == True
```

Comment:
A user won't call this. What's the method/function in public API that should be called to get this one executed, that should be used here.

Reply:
```suggestion
    assert e1.__contains__([7, 213, 1])
```